code/game/machinery/vendors/vending.dm

```diff
@@ -420,6 +420,9 @@
 		return
 
 	if(isliving(AM) && prob(aggressive_tilt_chance))
+		var/mob/living/to_be_tipped = AM
+		if(to_be_tipped.incorporeal_move) // OooOooOoo spooky ghosts
```

Comment:
I don't think this is (strictly) where you want this to be, I think you'd also want to have it in the loop checking for things on the tile in the tilt logic so they don't get damaged by vendors tilting on their tiles.

Reply:
Will do!